HLTrigger/Timer/plugins/FastTimerService.cc

```diff
@@ -767,7 +767,7 @@ void FastTimerService::PlotsPerJob::fill_lumi(AtomicResources const& data, unsig
 ///////////////////////////////////////////////////////////////////////////////
 
 FastTimerService::FastTimerService(const edm::ParameterSet& config, edm::ActivityRegistry& registry)
-    : tbb::task_scheduler_observer(true),
+    : tbb::task_scheduler_observer(),
       // configuration
```

Comment:
so
```suggestion
    :  // configuration
```

Reply:
There are two functions that are inherited from task_scheduler_observer according to comment in the file.

Reply:
the change does not remove the inheritance, only the (unnecessary) explicit call to the default constructor.